sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -562,7 +562,7 @@ class EthService(
       null,
       null,
       parentInfo.withdrawalEpochInfo,
-      None,
+      parentInfo.vrfOutputOpt, // same as for parent block, used as a source of BlockContext.random field
```

Comment:
Did you verify this with Ethereum? `Hash.ZERO` would also make sense I think, as the real data is unknown at this point.

Reply:
in Ethereum `pending` tag is forbidden for the places where we want to use it.

Reply:
after discussion: ok to be in this way, we need some real random value, not Zeros